src/app/modules/verification/__tests__/verification.controller.spec.ts

```diff
@@ -651,6 +670,7 @@ describe('Verification controller', () => {
         recipient: MOCK_ANSWER,
       })
       expect(mockRes.sendStatus).toHaveBeenCalledWith(StatusCodes.CREATED)
+      expect(disableSpy).toBeCalledWith(MOCK_FORM)
     })
 
     it('should return 400 when form SMS parameters are malformed', async () => {
```

Comment:
need add test cases for when `checkFreeSmsSentByAdminAndDeactivateVerification` returns an error

Reply:
yep, i thought about this but the only observable behaviour which we could test is the logging - is that what you meant?